Results_Engine/Query/DisplayMeshResults.cs

```diff
@@ -82,17 +85,16 @@ public static partial class Query
 
             if (gradientOptions == null)
                 gradientOptions = new GradientOptions();
-            
-            //Check if no identifier has been provided. If this is the case, identifiers are searched for in the objects
-            identifier = meshes.First().FindIdentifier(identifier);
 
             List<IMesh<TNode, TFace>> meshList = meshes.ToList();
 
             // Map the MeshResults to Meshes
-            List<List<IMeshResult<IMeshElementResult>>> mappedResults = meshList.MapResults(meshResults, "ObjectId", identifier, caseFilter);
-            Func<IMeshElementResult, double> propertyFuction = GetPropertFunc(meshResults.First().Results.First() as dynamic, meshResultDisplay);
+            List<List<IMeshResult<IMeshElementResult>>> mappedResults = meshList.MapResults(meshResults, "ObjectId", objectIdentifier, caseFilter);
+            //Get function for extracting property from results
+            var resPropSelectorAndQuantity = meshResults.First().ResultItemValueProperty(meshResultDisplay);
```

Comment:
To aid reading the code and remembering what is inside the Tuple, I generally like to use names with underscores e.g.
name_propSelector_quantity, but that's my personal preference